.github/actions/devcontainer_run_command/action.yml

```diff
@@ -103,6 +103,9 @@ inputs:
     description: "The AppService plan sku used by tests"
     required: false
     default: ""
+  RESOURCE_PROCESSOR_NUMBER_PROCESSES_PER_INSTANCE:
```

Comment:
Could also name it RESOURCE_PROCESSOR_PARALLELISM to make it slightly less of a mouthful

Reply:
yes, could have... but I think the long name is ok and also explains what it does :)